.github/workflows/build.yml

```diff
@@ -61,14 +61,13 @@ jobs:
           asset_path: ./devonfw_guide.pdf
           asset_name: devonfw_guide.pdf
           asset_content_type: application/pdf
-      - name: Update Website on success
-        run: |
-          git clone -b develop https://${BUILD_USER}:${BUILD_USER_PASSWD}@github.com/${ORG}/devonfw.github.io.git --recurse-submodules
+     - name: Update Website 
+       run: |
+          git clone https://${BUILD_USER}:${BUILD_USER_PASSWD}@github.com/${ORG}/devonfw.github.io.git --recurse-submodules
           cd devonfw.github.io/devonfw-guide
           git checkout master
           git pull
           cd ..
-          git checkout develop
           git add devonfw-guide
           git commit -m "[DocUpdate]"
-          git push origin develop
+          git push origin develop
```

Comment:
why can you push to develop if you don't know the default branch?
The previous was assuring that develop branch is checked out

Reply:
develop is the default branch for the website.